portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,214 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Edson's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+     <link rel="stylesheet" href="style.css">
+    <title>Edson Casimiro</title>
+    <!--adding google fonts and fontawesome-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css">
+</head>
+<body>
+
+    <nav>
+        <div class="logo">
+            Edson Casimiro
+        </div>
+        <div class="hamburger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+        </div>
+        <ul class="nav-links">
+            <li><a href="#home">Home</a></li>
+            <li><a href="#course">Courses&skills</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#project">Project</a></li>
+
+        </ul>
+    </nav>
+<p id="home"></p>
+
+
+    <div class="start" style="overflow:auto;">
+        <div class="intro1" >
+            <img src="me.jpg" alt="edson casimiro">
+            <h3>Computer Engineering Student</h3>
+            <p><span><i class="material-icons" id="btn">home</i>Ks, Angola</span> <span><i class="material-icons"
+                        id="btn">location_on</i>Istanbul, Turkey</span></p>
+            <p><span><i class="material-icons" id="btn">translate</i></span>Portuguese, English, Turkish </p>
+
+            <br>
+            <h1>
+                <a href="https://www.linkedin.com/in/edson-casimiro-435500137/"
+                    style="textdecoration:none; color:black;">
+                    <i class="fab fa-linkedin"></i>
```

Comment:
One thing to optionally consider is how this will work with screen readers. Will a blind user be able to interact with these icons?

(It's up to you whether you want to focus on that, but it's something we have to think about a lot in industry.)

Reply:
hum, i see. how do screen readers identify icons tag? Or what could be the best way to apply then on the website?
Instead of icons,  is it recommended to just use simply <a>tag</a> ?